fetch/stale-while-revalidate/fetch.html

```diff
@@ -17,9 +17,12 @@
   var request_token = token();
 
   const response = await fetch(`resources/stale-script.py?token=` + request_token);
+  const body = await response.text();
```

Comment:
Can you add a comment above this line describing why it's being added?

Reply:
Done.